style.css

```diff
@@ -813,4 +815,603 @@ justify-content: space-between;
+    }
+
+    .about-car__text {
+        margin-top: 25px;
+    }
+
+    .description__advantage-wrapper {
+    position: absolute;
+    top: 55px;
+    right: 68px;
+    width: 190px;
+    min-height: 250px;
+    margin-left: 55px;
+    }
+    
+    .advantage__figures_size_medium {
+    font-size: 35px;
+    line-height: 43px;
+    }
+
+    .description__order {
+        position: relative;
+        bottom: 0;
+        right: 0;
+        min-width: 420px;
+        padding-left: 40px;
+        margin: 5px auto 0;
+    }
+    .order__know-more {
+        font-size: 16px;
+        line-height: 20px;
+    }
+
+
+    .order__know-more::before {
+        background-size: 32px;
+        left: 0;
+        width: 32px;
+        height: 32px;
+    }
+
+    .order__button {
+        font-size: 16px;
+        padding-right: 11px;
+        width: 220px;
+        height: 55px;
+    }
+
+    .order__button_border_white {
+        border: 3px solid #FFFFFF;  
+    }
+}
+
+@media only screen and (min-width: 500px) and (max-width: 679px) {
+
+    .description__advantage-wrapper {
+        display: none;
+    }
+
+    .description {
+        width: 100%;
+        min-height: 550px;
+        background-image: url("img/tesla-model-s-cockpit-ipad.png");
+        background-size: 809px auto;
+        background-repeat: no-repeat;
+        background-position-x: 0;
+        background-position-y: 23px;
+       
+    }
+    .description__title {
+        position: absolute;
+        margin: 0;
+        right: 50px;
+        top: 30px;
+    }
+    .description__main-wrapper {
+        position: absolute;
+        left: 0px;
+        top: 525px;
+        max-width: 809px;
+        width: 100%;
+        min-height: 437px;
+        padding-left: 0;
+        padding-top: 51px;
+    }
+
+    .description__about-car {
+        margin: 0 auto;
+        width: 316px;
+    }
+
+    .description__order {
+        position: relative;
+        padding-left: 0;
+        margin: 0 auto;
+        bottom: 0;
+        right: 0;
+    }
+
+    .order__know-more {
+        padding-left: 6px;
+    }
+    .order__know-more_size_big {
+        font-size: 16px;
+    }
+    .order__know-more::before {
+        top: 14px;
+    }
+}
+
+@media only screen and (max-width: 499px) {
+    .description__title,
+    .description__advantage-wrapper,
+    .description__order {
+        display: none;
+    }
+    .description {
+        width: 100%;
+        min-height: 300px;
+        background-image: url("img/tesla-model-s-cockpit-ipad.png");
+        background-position-x: -30px;
+        background-position-y: -35px;
+    }
+    .description__main-wrapper {
+        position: absolute;
+        left: 0px;
+        top: 275px;
+        max-width: 809px;
+        width: 100%;
+        min-height: 285px;
+        padding-left: 0;
+        padding-top: 22px;
+    }
+
+    .description__about-car {
+        margin: 0 auto;
+        width: 316px;
+    }
+    .about-car__text {
+        margin-bottom: 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Angleter_tesla/pullRequest?prid=5388479)